src/python/rebalance/rebalance_rackaware.py

```diff
@@ -444,7 +439,8 @@ def generate_reassignment_plan(plan_directory, topics, brokers_info, compute_sto
     if compute_storage_cost:
         partitions_sizes = get_storage_info(host_info)
         logger.debug("Partition Sizes: %s", str(partitions_sizes))
-    logger.debug("Host Information: %s", str(host_info))
+    logger.debug("\nHost Information: \n%s", host_info)
+    logger.info("\n\nHost Rack Information: \n%s\n", "\n".join(list(map(lambda datum: "Broker: {0} Rack: {1}".format(str(datum['brokerId']), datum['rack']), host_info))))
```

Comment:
Should we use debug instead of info here?

Reply:
changed